app/src/main/java/cs465/illinois/edu/dogonthequad/CreateMeetupSelectDogsActivity.java

```diff
@@ -45,11 +48,12 @@ protected void onCreate(Bundle savedInstanceState) {
     //temporary, should use the active user's list of dogs
 
     public List<Dog> GetDogs() {
-        ArrayList<Dog> dogs = new ArrayList<Dog>();
+        ArrayList<Dog> dogs = API.getDogs();
```

Comment:
Guess it's not a big deal right now but we should probably only get dogs that are associated with the current user. Perhaps we can write something into the API to do that